README.md

```diff
@@ -11,12 +11,14 @@ Put files to be populated by Ignition in the tree rooted at
 `data/ignition/files/`. Put systemd units in `data/ignition/systemd/units`.
 These are all built into the binary at compilation time.
 
-The ZTP manifests provided by the users are read from `./manifests`.
+The ZTP manifests provided by the users are read from `data/ignition/files/etc/manifests.`
```

Comment:
Out current process of using `go run` to run the tool is obscuring the fact that the files getting automatically added to the ignition (by virtue of being present in `data/ignition/files` are available because they are being built into the golang binary before it runs.
In general though, we can't rely on compile-time presence of the manifests because they have to be provided by the user. We need to explicitly search the manifests directory at runtime and add those files to the ignition.

Reply:
Good point. I have a better understanding of how the embed package is being used now. I will move the manifests out of data/ignition and have them read during runtime.